File: docs/exemplos/py/menu.py
while True:
    print("MENU--------------")
    print("1 - listar produtos")
    print("2 - cadastrar produtos")
    print("3 - editar produtos")
    print("4 - deletar produtos")
    print("0 - sair")
    opção = int(input("\t>>> "))

    if opção == 1:
        print("\t>>> ", "Listando os produtos. . .")
    elif opção == 2:
        print("\t>>> ", "Vamos cadastrar um novo produto. . .")
    elif opção == 3:
        print("\t>>> ", "Vamos escolher um produto para editar. . .")
    elif opção == 4:
        print("\t>>> ", "Vamos escolher um produto para deletar. . .")
    elif opção == 0:
        print("\t>>> ", "Obrigado. Até breve!")
        break
    else:
        print("\t>>> ", "Opção inválida!")

File: docs/exemplos/c/menu.c
#include <stdio.h>


void main(){
int opcao;
do{
    printf("MENU--------------\n");
    printf("1 - listar produtos\n");
    printf("2 - cadastrar produtos\n");
    printf("3 - editar produtos\n");
    printf("4 - deletar produtos\n");
    printf("0 - sair\n");
    printf("\t>>> \n");

    scanf("%d", &opcao);

    switch(opcao){
        case 1:
            printf("\t>>> Listando os produtos. . .\n");
            break;
        case 2:
            printf("\t>>> Vamos cadastrar um novo produto. . .\n");
            break;
        case 3:
            printf("\t>>> Vamos escolher um produto para editar. . .\n");
            break;
        case 4:
            printf("\t>>> Vamos escolher um produto para deletar. . .\n");
            break;
        case 0:
            printf("\t>>> Obrigado. Ate breve!\n");
            break;
        default:
            printf("\t>>> Opcao invalida!\n");
        }
    }while(opcao != 0);
}
